### ISSUE TITLE
can't start wss in android (termux).

### ISSUE THREAD

**Issue by X7md** [vmware-labs/wasm-workers-server#25]
Hi I'm try using https://termux.dev to run this, but I'm facing this...

```sh
/wws . 
⚙️  Loading routes from: .
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 22, kind: InvalidInput, message: "Invalid argument" }', /home/ereslibre/.cargo/registry/src/github.com-1ecc6299db9ec823/wasmtime-jit-1.0.1/src/code_memory.rs:63:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace```

**Comment by ereslibre** [comment]
Thank you @X7md for the report! Can you please run the `wws` command with `RUST_BACKTRACE` exported to `1` or `full` and provide the reported backtrace?

https://github.com/bytecodealliance/wasmtime/issues/4972 seems related to this issue at first glance.

**Comment by X7md** [comment]
With RUST_BACKTRACE=full

```sh
~/WebAssembly/test $ RUST_BACKTRACE=1RUST_BACKTRACE=full ./wws .
⚙️  Loading routes from: .
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 22, kind: InvalidInput, message: "Invalid argument" }', /home/ereslibre/.cargo/registry/src/github.com-1ecc6299db9ec823/wasmtime-jit-1.0.1/src/code_memory.rs:63:14
stack backtrace:
   0: rust_begin_unwind
             at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/panicking.rs:584:5
   1: core::panicking::panic_fmt
             at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/core/src/panicking.rs:142:14
   2: core::result::unwrap_failed
             at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/core/src/result.rs:1805:5
   3: wasmtime_jit::instantiate::CompiledModule::from_artifacts                                                          4: wasmtime::module::Module::from_parts
   5: wasmtime::module::Module::from_binary
   6: wws::runner::Runner::new
   7: wws::router::initialize_routes
   8: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
   9: std::thread::local::LocalKey<T>::with
  10: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
  11: tokio::runtime::scheduler::current_thread::Context::enter
  12: tokio::macros::scoped_tls::ScopedKey<T>::set
  13: tokio::runtime::scheduler::current_thread::CurrentThread::block_on
  14: tokio::runtime::Runtime::block_on
  15: wws::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
~/WebAssembly/test $ ls
index.js  wws
```

**Comment by X7md** [comment]
Yeah, it's `wasmtime` bug or something...

I test it on android emulator (Android 12, x86_64)

![image](https://user-images.githubusercontent.com/54203033/201328180-7a78fb8f-b546-497a-99ea-55c45abd96c9.png)

and works fine.

**Comment by X7md** [comment]
Solved in `0.6.0` ✅ 

![Screenshot_20221203-234742_One UI Home](https://user-images.githubusercontent.com/54203033/205461567-40450d69-7206-47ea-b21d-d105d2dfe9ea.jpg)
